### DISCUSSION TITLE
The problem of setting the LSTM hidden_size

### DISCUSSION THREAD

**Discussion by aiqinxuancai** [xamarin/TorchSharp#487] [Q&A]
I am trying to translate a pytorch script, and since I don't use much for pytorch and torchsharp, I can't determine the cause of this problem.
It seems to be prompting me to pass in a hiddenSize of 0, but I passed in 256 when I created it.

![image](https://user-images.githubusercontent.com/4475018/144566180-c6b88a7a-9e8c-445b-a664-dde3b40494e5.png)

![image](https://user-images.githubusercontent.com/4475018/144565642-350d9991-e132-4bc8-b609-6ee0ae779d98.png)

**Comment by NiklasGustafsson** [comment]
@aiqinxuancai -- if this is still a problem, please file a bug.
